docs/components/plugins/Video.tsx

```diff
@@ -46,7 +46,7 @@ type VideoProps = React.PropsWithChildren<{
 
 const Video = ({ controls, spaceAfter, url, file, loop = true }: VideoProps) => {
   const [hover, setHover] = useState(false);
-  const [forceShowControls, setForceShowControls] = useState(false);
+  const [forceShowControls, setForceShowControls] = useState(url?.includes('youtube.com'));
```

Comment:
Will this logic handle youtu.be links? I'm sure the `react-player` will internally load the video just fine, but I'm wondering if the controls will still be shown if we're relying on the explicit "youtube.com" string from the `url` prop for that 🤔

Reply:
We are currently not using the shortened YT links (and an URL need to be provided manually by editor), but probably we can add this additional domain to the check.

@fobos531 Are you able to add code suggestion in this PR, if not, would you like to make a follow up PR adding that check? 🙂